execution-plane/arcon/src/stream/node/mod.rs

```diff
@@ -330,13 +336,8 @@ where
                         self.state.current_epoch.set(&mut *self.state_backend, e)?;
 
                         // call handle_epoch on our operator
-                        let _operator_state = self.operator.handle_epoch(
-                            e,
-                            OperatorContext::new(
-                                &mut self.channel_strategy,
-                                &mut *self.state_backend,
-                            ),
-                        );
+                        // TODO this seems useless?
+                        let _operator_state = self.operator.handle_epoch(e, make_context!(self));
```

Comment:
Yeah, `handle_epoch` doesn't need to return anything since a `state_backend` is passed to it anyway. It should probably have the same signature as `handle_watermark`. This is a remnant of earlier design which I forgot to fix.